frontend/src/components/landing/HighlightedFeatures.tsx

```diff
@@ -0,0 +1,103 @@
+import { useL10n } from "../../hooks/l10n";
+import CreateUnlimitedEmailMasksImage from "./images/highlighted-features/features-unlimited-email-masks.svg";
+import CreateMasksOnTheGoImage from "./images/highlighted-features/features-instantly-masks-on-the-go.svg";
```

Comment:
Question: This image has unlocalized text in it. Is there a design where we can remove this? 

<img width="265" alt="image" src="https://user-images.githubusercontent.com/2692333/218777846-fdce09b1-3f25-437c-afba-53e73c50e259.png">